plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -161,7 +161,7 @@ public void write(CheckpointEntries entries, TrinoOutputFile outputFile)
         }
         List<DeltaLakeColumnHandle> partitionColumns = extractPartitionColumns(entries.metadataEntry(), entries.protocolEntry(), typeManager);
         List<RowType.Field> partitionValuesParsedFieldTypes = partitionColumns.stream()
```

Comment:
Let's add a corresponding test in `io.trino.plugin.deltalake.transactionlog.checkpoint.TestCheckpointWriter`

Reply:
I intentionally avoided that. Both TestCheckpointWriter & TestCheckpointEntryIterator are not suitable to verify `partitionValues_parsed` field because `AddFileEntry` doesn't hold the value.

Reply:
I was thinking about a test similar to  `io.trino.plugin.deltalake.transactionlog.checkpoint.TestCheckpointEntryIterator#testReadAddEntriesPartitionPruning` with corresponding resource files